composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.4.1",
+        "php": "^7.4.1 || ^8.0",
```

Comment:
So the plan is to drop 7.x in the next iteration? If not, how will the compatibility be guaranteed when CI is only run with one version? :thinking: @akondas?